spec/models/order_spec.rb

```diff
@@ -1,5 +1,73 @@
 require 'rails_helper'
 
 RSpec.describe Order, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  
+  let(:user) { User.create(full_name: "John Doe",
+                           user_name: "john",
+                           password: "password",
+                           email: "john@johndoe.com") }
+  
+  let(:category) { Category.create(name: "new cat") }
+  
+  let(:item) { Item.create(title: "Unicorn Pie",
+                           description: "Good",
+                           price: 8.00,
+                           status: "active",
+                           categories: [category]) }
+  
+  let(:valid_attributes) { { status: "ordered",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.